FRINXio/frinx-frontend
# FR-317 mpls topology version

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FR-317` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

test with https://github.com/FRINXio/frinx-inventory-server/pull/464

add MplsDevice to arango db, you should see green node when `Compare current topology with:`
